src/main/kotlin/no/skatteetaten/aurora/boober/platform/AbstractHandler.kt

```diff
@@ -0,0 +1,27 @@
+package no.skatteetaten.aurora.boober.platform
```

Comment:
This should probably be under mapper, and not directly under boober. One may argue that AuroraConfig, DeploymentSpec and related mapping logic perhaps deserves its own "top level" aurora package, like no.skatteetaten.aurora.auroraconfig? When/if modularizing boober, we could even consider extracting it into its very own repo.